### PR TITLE
Add functionality for Closest Police Station

### PR DESCRIPTION
**Pull request by chrislreich**
Currently returns a dummy value because we are not yet interacting with ArcGIS API. If the user asks the "town of brookline" for the closest police station, we will prompt for an address and then return said dummy value. Up next is to pass this address to some ArcGIS service and actually return the closest police station.

### CODE REVIEW

File: brooklinevoiceapp/mycity/intents/intent_constants.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Constants used across intents"""
+  
+# The key used for the current address in session attributes
+CURRENT_ADDRESS_KEY = "currentAddress"
+ZIP_CODE_KEY = "Zipcode"
```
[the remark this answers is not in the document]
Comment (chrislreich) [reply brooklinevoiceapp/mycity/intents/intent_constants.py]: Legacy code from voiceapp311, changed to "zipcode"

File: brooklinevoiceapp/mycity/intents/police_station_intent.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseDataModel object
+    """
+    
+    response = MyCityResponseDataModel()
+    set_address_in_session(mycity_request)
+    response.session_attributes = mycity_request.session_attributes
+    current_address = \
+            mycity_request.session_attributes.get(intent_constants.CURRENT_ADDRESS_KEY)
+    if current_address == None:
```
[the remark this answers is not in the document]
Comment (chrislreich) [reply brooklinevoiceapp/mycity/intents/police_station_intent.py]: Fixed

File: brooklinevoiceapp/brookline_controller.py [hunk 3]
```diff
@@ -11,6 +12,30 @@ def execute_request(mycity_request):
...
+def on_intent(mycity_request):
+    """
+    If the event type is "request" and the request type is "IntentRequest",
+    this function is called to execute the logic associated with the
+    provided intent and build a response. Checks for required
+    session_attributes when applicable.
+
+    :param mycity_request: MyCityRequestDataModel object with
+        request_type IntentRequest
+    :return: MyCityRequestDataModel object corresponding to the intent_name
```
[the remark this answers is not in the document]
Comment (chrislreich) [reply brooklinevoiceapp/brookline_controller.py]: Legacy comment from voiceapp311. Fixed

File: brooklinevoiceapp/interaction_models/en_US.json [hunk 4]
```diff
@@ -24,16 +24,60 @@
...
                     "samples": [
-                        "Hello",
-                        "Are you there",
-                        "Hi Brookline"
+                        "closest police",
+                        "police near me",
+                        "police station near me",
+                        "police station",
+                        "Closest police stations",
+                        "Where is the closest Police Stations"
```
[the remark this answers is not in the document]
Comment (jaumb) [reply brooklinevoiceapp/interaction_models/en_US.json]: I added a few in the followup to this PR: /brooklinevoiceapp/interaction_models/en_US.json if that so we could skip them in this PR

[the remark this answers is not in the document]
Comment (chrislreich) [reply brooklinevoiceapp/interaction_models/en_US.json]: Added, good idea

File: brooklinevoiceapp/mycity/intents/police_station_intent.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    Finds the closest police station in Brookline
+    to a given address
+
+    :param mycity_request: MyCityRequestDataModel object
+    :return: MyCityResponseDataModel object
+    """
+    
+    response = MyCityResponseDataModel()
+    set_address_in_session(mycity_request)
+    response.session_attributes = mycity_request.session_attributes
```
[the remark this answers is not in the document]
Comment (chrislreich) [reply brooklinevoiceapp/mycity/intents/police_station_intent.py]: Saving this discussion for later because more of a design decision for the larger group

**Pull request merged**